conf/flight_plans/joystick.xml

```diff
@@ -3,7 +3,18 @@
 <flight_plan alt="75" ground_alt="0" lat0="43.46223" lon0="1.27289" max_dist_from_home="1500" name="Basic" security_height="25">
   <header>
 #include "subsystems/datalink/datalink.h"
-#include "firmwares/fixedwing/joystick.h"
+
+#ifdef NAV_C
+#include "firmwares/fixedwing/stabilization/stabilization_attitude.h"
+#include "autopilot.h"
+void joystick_handler(void) {
```

Comment:
First I forgot to update this one for ABI message.
But the idea was to avoid making a module only to implement this function and also to show a way to make a fully integrated flight plan that load modules, define variables and callback, etc